stacks/syrah/syrah.yaml

```diff
@@ -154,6 +154,7 @@ serial_packages:
               amd: +rocm ~cuda
     - intel-oneapi-tbb
     - jasper
+    - kallisto +hdf5 ^hdf5 ~mpi
```

Comment:
When looking at my spack 0.18 I don't see a +hdf5 variant on kallisto. Are you using a more recent spack?

Reply:
yes I did a `spack clone` using the spack from syrah hopping it clones the same version